examples/tests/requirements.txt

```diff
@@ -1,7 +1,8 @@
 # pytest 6.0 has linter-breaking changes
 pytest>=6.0.1
-tensorflow==2.2.0
+tensorflow==2.2.1
 torch==1.4.0
 torchvision==0.5.0
 pandas==1.0.3
 tensorflow_datasets
+scipy
```

Comment:
what's this for?

Reply:
We were failing tests w/o it, my guess is that dependencies of TF changed from 2.2.0 -> 2.2.1 where it was no longer installed `scipy`.